### DISCUSSION TITLE
Waypoint placement

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#37] [Q&A] [answered]
A function that places the waypoints exactly on the road would not be bad.
The BRouter goes sometimes on strike if the waypoints are not exactly on the road.

**Comment by devemux86** [comment, marked as answer]
Do you mean automatically snap waypoints on the roads?
The road network is not known before route is calculated.

Placement of waypoints is done with the crosshair action,
having control over zoom / move for a precise placement.

There is the extra "Place waypoints on street" function.
(long press on the calculated route)

**Comment by devemux86** [comment]
> The BRouter goes sometimes on strike if the waypoints are not exactly on the road.

Indeed, it seems more restrictive than other routers in this case.
Or is this affected by the BRouter profile configuration?

**Reply by Leeb64** [reply]
Sometimes the WP can be 20-30m next to the road and it works... other times the WP has to be in the middle of the road otherwise it will complain.

**Reply by devemux86** [reply]
Can you also reproduce this on the [BRouter website](https://brouter.de/brouter-web/)?

**Reply by Leeb64** [reply]
It seems to work on BRouter Web.
With the Cruiser app it usually works (about 90%).....not always

**Reply by devemux86** [reply]
Can you write what options / profile do you use on BRouter website?
And what options in Cruiser + BRouter apps?

Also a route sample to test?

**Comment by Leeb64** [comment]
[0_KurvigERsatz.txt](https://github.com/devemux86/cruiser/files/10768360/0_KurvigERsatz.txt)

It's this BRouter profile.
Replace the txt extension with brf
